File: Zestawy Zadań/Python/3 - Iteracje, Pętle/Zestaw 2 Zadania/1.3.py
# Pobranie wyrazu od użytkownika
wyraz = input("Podaj wyraz: ")

# Pętla do-while, która sprawdza, czy wyraz jest równy "Jan" lub "Janek" i pobiera go ponownie, jeśli nie
while wyraz != "Jan" and wyraz != "Janek":
  print("Podany wyraz jest niepoprawny. Podaj ponownie: ")
  wyraz = input()

# Wyświetlenie komunikatu "Cześć" i wyrazu
print("Cześć", wyraz)

File: Zestawy Zadań/C++/3 - Iteracje - Pętle/Zestaw 2 zadania/1.3.cpp
// Napisz program, który:
// a.Pobiera od użytkownika wyraz
// b.Jeśli pobrany wyraz jest różny od „Jan” oraz od „Janek”, to program ponownie pobiera
// wyraz(skok do punktu a)
// c.Program wypisuje „Cześć” i w tym samym wierszu podany wyraz.
// Zastosuj pętlę do – while.

#include <iostream>
#include <string>

using namespace std;

int main()
{
    string a;
    do
    {
        cin >> a;
    } while (a != "Jan" && a != "Janek");
    cout << "Czesc" << a << endl;
    return 0;
}

